### ISSUE TITLE
Make CMake to automatically init submodules

### ISSUE THREAD

**Issue by xtracko** [crocs-muni/oneclick#8]
Instead of manually typing

```
git submodules init
git submodules update
```

make CMake to automate the process.

**Comment by mukrop** [comment]
Isn't this already done? I don't want to close the issue myself.

**Comment by xtracko** [comment]
No. I don't think that this is even possible in older versions of CMake or it is extremely difficult. The current version of stable cmake (version 3.6) has some support for this feature, but it is too early for the change. So leave this issue open.
